src/app/components/core/user-management/user-profile/user-profile.component.ts

```diff
@@ -80,17 +80,19 @@ export class UserProfileComponent implements OnInit {
   }
 
   getAccessMethod(accessMethod) {
+    let method;
```

Comment:
accessMethod: string
and
method: string

Reply:
We can improve on this later.